src/resolve-dependency.ts

```diff
@@ -95,15 +107,19 @@ function getExportsTarget (exports, conditions, cjsResolve) {
       }
     }
   }
-  else if (exports === null) {
-    return exports;
-  }
+
+  return null;
 }
 
-function resolveExportsTarget (pkgPath, exports, subpath, job, cjsResolve) {
-  if (typeof exports === 'string' ||
-      typeof exports === 'object' && !Array.isArray(exports) && Object.keys(exports).length && Object.keys(exports)[0][0] !== '.')
-    exports = { '.' : exports };
+function resolveExportsTarget (pkgPath: string, exp: string | { [key: string]: string }, subpath: string, job: Job, cjsResolve: boolean): string | undefined {
```

Comment:
I noticed that `getExportsTarget()` handles the `exports: null` case however this `resolveExportsTarget()` function does not. Should we also handle `exports: null` here?

Reply:
Yes on 117 we should probably have a null check too for consistency.

Reply:
What should happen for `null` on line 117?

- `return null`?
- assign `exports: { '.' : null }`?
- assign `exports: null`?
- something else?

Reply:
Actually `exports` can never be `null` or `undefined` here as it is guarded in all calls.

Reply:
Good point, I added a `PkgCfg` and `Exports` type so we can confirm all cases are covered, thanks!